.changeset/platform-sandbox-sidecar-probe.md

```
@@ -0,0 +1,9 @@
+---
+'@mastra/platform-workspace': patch
+---
+
+Defer address registry population until sidecar /health probe succeeds.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)